muspinsim_input/muspinsim_input.xml

```diff
@@ -35,46 +35,49 @@
-                        <option value="mu">mu</option>
+                    <param name="spin_preset" type="select" value="mu" label="Species" help="Select 'custom' to define own">
+                        <option selected="true" value="mu">mu</option>
                         <option value="e">e</option>
-                        <option selected="true" value="custom">custom</option>
+                        <option value="custom">custom</option>
                     </param>
                     <when value="custom">
-                        <param name="spin" optional="false" type="text" label="Enter custom species"/>
-                        <param name="atomic_mass" optional="true" type="float" value="" label="Custom Atomic Mass, Leave blank to use default symbol mass"/>
+                        <param name="spin" optional="false" type="text" label="Species name"/>
+                        <param name="atomic_mass" optional="true" type="integer" min="0" value="" label="Atomic mass" help="Leave blank to use default mass - whole numbers only"/>
                     </when>
                     <when value="mu"/>
                     <when value="e"/>
                 </conditional>
             </repeat>
         </section>
-        <section name="interaction_params" expanded="true" title="Spin Interaction Parameters">
-            <repeat name="interactions" title="Interactions to simulate">
+        <section name="interaction_params" expanded="true" title="Spin Interactions" help="Interaction terms available: zeeman, hyperfine, dipolar, quadrupolar or dissipation">
+            <repeat name="interactions" title="Interactions to simulate" help="Add couplings between spins, and/or dissipation terms, see muspinsim docs for more info">
                 <conditional name="interaction_options">
                     <param name="interaction" type="select" label="Choose interaction type">
-                        <option value="zeeman">zeeman</option>
+                        <option value="zeeman">Zeeman</option>
                         <option value="hyperfine">hyperfine</option>
                         <option value="dipolar">dipolar</option>
                         <option value="quadrupolar">quadrupolar</option>
                         <option value="dissipation">dissipation</option>
                     </param>
                     <when value="zeeman">
-                        <param name="zeeman_index" type="integer" value="" label="Index of coupled spin (index starts from 1)"/>
-                        <param name="zeeman_vector" type="text" value="" label="Vector for Zeeman coupling (Tesla)"
-                            help="Define 1X3 vector for local magnetic field coupling.
+                        <param name="zeeman_index" type="integer" value="" label="Index of coupled spin" min="1"
+                               help="Index refers to the position of the spin as selected in 'spins' section "/>
```

Comment:
This is a tricky one to phrase. I think we definitely need to mention the 1-indexing though. What do you think of this?
```suggestion
                               help="Index refers to the order of the spins listed in 'spins' section.  The first spin in the list has index 1, the second has index 2, and so on. "/>
```

Reply:
Thanks for accepting the suggestion. This still needs updating in all the other interaction types too - see the macro idea above